### PR TITLE
add changes made to the BIDS tables for the mnc2bids conversion script

### PR DESCRIPTION
**Pull request by cmadjar**
## Brief summary of changes

This PR modifies the BIDS tables used to convert MINC datasets into BIDS datasets. In addition, it adds tables to store the path to the created files under the BIDS structure in the database so later on it can be queried via the API.

- Table modification (in green) required for the BIDS protocol identification and file naming scheme
BIDS_protocol_and_filename_mapping

- New tables (in green) where the converted BIDS files will be located
BIDS_converted_files_table_structure

#### Testing instructions

1. Ensure the SQL patch in the `SQL/New_patches` are correctly executing
2. Ensure the schema `SQL/0000-00-00-schema.sql` is loading properly
3. Ensure the `SQL/9999-99-99-drop_tables.sql` is properly cleaning all the BIDS tables

#### Link(s) to related issue(s)

* Required by the changes made in https://github.com/aces/Loris-MRI/pull/636

### CODE REVIEW

**Comment by cmadjar** [comment]
This is blocked by https://github.com/aces/Loris/pull/7475 which fixes the `9999-99-99-drop_tables.sql` bug in the file.

**Comment by kongtiaowang** [comment]
haha. The error has been fixed by git action itself.

**Comment by cmadjar** [comment]
@PapillonMcGill any chance you could review this PR? Thank you :)

File: SQL/0000-00-00-schema.sql
Comment (PapillonMcGill) [inline SQL/0000-00-00-schema.sql:911]: @cmadjar What is the reason the name of the attribute is `BIDSExportFileLevelID` in this table and refer to `BIDSExportFileLevelCategoryID`?

Comment (PapillonMcGill) [reply SQL/0000-00-00-schema.sql:911]: @cmadjar  ping!

Comment (cmadjar) [reply SQL/0000-00-00-schema.sql]: @PapillonMcGill Thank you! Corrected. Not sure why I did that in the first place...

**Review by PapillonMcGill — APPROVED** [review]
pending Travis approval.

File: SQL/New_patches/2021-06-11_exported_files_BIDS_tables.sql
Comment (cmadjar) [reply SQL/New_patches/2021-06-11_exported_files_BIDS_tables.sql]: @driusan good catch! I removed them in the next commit. Sorry about that...

**Pull request merged**
